make-it/EmMakefile

```diff
@@ -92,11 +92,11 @@ ifeq ($(TARGET), asmjs-unknown-emscripten)
 endif
 
 ifeq ($(TARGET), wasm32-unknown-emscripten)
-   TARGET_FLAG = -s WASM=1 -s BINARYEN_TRAP_MODE=js -s ALLOW_MEMORY_GROWTH=1
+   TARGET_FLAG = -s WASM=1 -s ALLOW_MEMORY_GROWTH=1
 endif
 
 # NO_DYNAMIC_EXECUTION=1 Allows Vireo to be used in CSP environments where eval() is forbidden: https://github.com/kripken/emscripten/blob/master/src/settings.js#L563
-EM_OPT = $(EM_OPTFLAGS) $(TARGET_FLAG) -s NO_EXIT_RUNTIME=1 -std=c++14 -fno-exceptions -Werror --memory-init-file 0 -s NO_DYNAMIC_EXECUTION=1 -s MODULARIZE=1 -s EXPORT_NAME=VireoCreateCoreModule -s EXPORT_ES6=1 -s USE_ES6_IMPORT_META=0 --minify 0 -s NO_FILESYSTEM=1 -s ABORTING_MALLOC=0
```

Comment:
Looks like [NO_EXIT_RUNTIME=1 became the default](https://github.com/emscripten-core/emscripten/blame/a8a0d8e79ed5d4aa640e18ae7497ef4c48e5beaa/ChangeLog.md#L1464) was it removed because it's not needed or was it causing some other error?

Reply:
Yes. Now it is the default setting so removed it.